client/my-sites/patterns/components/category-gallery/style.scss

```
@@ -48,12 +48,23 @@
 
 	.patterns-category-gallery__item-preview--mirrored {
 		.patterns-category-gallery__item-preview-inner {
+			// necessary to trigger browser's repainting process to fix the appearance issue in FF and Safari
```

```suggestion
			// This is needed to make iframe lazy loading work in Firefox and Safari, see https://wp.me/pdtkmj-2wZ#comment-4707
```
It'd be nice to reference the P2 comment that mentions this